.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,35 @@
+name: Trivy
+
+on:
+  workflow_dispatch:
+
+jobs:
+  ubuntu-build:
```

Comment:
Can't we use this template here?
https://github.com/ballerina-platform/ballerina-library/blob/main/library-templates/generated-connector-template/files/.github/workflows/trivy-scan.yml

Reply:
Fixed with [69492b9e569069b5ab740b394085504a601f2728](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/1/commits/69492b9e569069b5ab740b394085504a601f2728)